examples/Makefile

```diff
@@ -1,18 +1,12 @@
-.PHONY: opencv object_detection delete minikube gatk hyperparameter
```

Comment:
1) Entirely removing the phony targets could trigger a bug, if there's a file with the name of any of these targets. It's unlikely, but I think worth covering.
2) You should set the shell on makefiles as well, like so: https://github.com/pachyderm/python-pachyderm/blob/0d464d83b381dbe0248eedf0f22fe9703678bedd/Makefile#L1

Reply:
First, thought long about my arguments vs. yours, and I'll walk back every one but kubectl, pachctl & echo.  The others I put in there in _anticipation_ of conflicts, and, you're right, the added complexity is not worth the anticipated need.

1. I tested for that; there aren't files with those names, just directories.  It works as we expect.
2. Great tip, thank you!

Reply:
new commit should address these issues.  Thanks for all your help making this better!